db/migrate/20200519073500_remove_visible_from_alchemy_pages.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class RemoveVisibleFromAlchemyPages < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :alchemy_pages, :visible, :boolean, default: false
```

Comment:
Can we `raise` here if any existing pages have `visible` set to `false`? Because that would indicate ppl have not run the upgrade task in 4.6, and will lose data.

Reply:
Sure thing

Reply:
@mamhoff added. Please re-review